ignite/metrics/ssim.py

```diff
@@ -100,7 +103,9 @@ def _uniform(self, kernel_size):
         return kernel.unsqueeze(dim=0)  # (1, kernel_size)
 
     def _gaussian(self, kernel_size, sigma):
-        kernel = torch.arange(start=(1 - kernel_size) / 2, end=(1 + kernel_size) / 2, step=1, dtype=torch.float32)
+        kernel = torch.arange(
```

Comment:
Here, i'd say:
```python
    ksize_half = (kernel_size - 1) * 0.5
    x = torch.linspace(-ksize_half, ksize_half, steps=kernel_size)
    pdf = torch.exp(-0.5 * (x / sigma).pow(2))
    kernel1d = pdf / pdf.sum()
```
may be a bit more efficient.

Reply:
Done!